vochain/scrutinizer/vote.go

```diff
@@ -290,7 +308,7 @@ func (s *Scrutinizer) addVoteIndex(nullifier, pid []byte, blockHeight uint32,
 			CreationTime: time.Now(),
 		})
 	}
-	return s.db.Insert(nullifier, &VoteReference{
+	return s.db.Insert(nullifier, &types.VoteReference{
```

Comment:
I think it makes more sense to declare the scrutinizer types inside the scrutinizer package. In the future we should try to get rid of the types.go root file.

Reply:
The problem here is that `types.MetaResponse` has a `Process` field, so with Process declared in Scrutinizer there is a circular import. I agree this fix isn't great, though. 

For now I'll move the other types back to scrutinizer and leave Process in types with a todo.

Reply:
We should check why the scrutinizer package needs types.go

Reply:
It needs types.go directly for HexBytes, VotePackage (for creating test votes), ScrutinizerOnProcessData, KeyKeeperMaxKeyIndex, and the api types EnvelopePackage, EnvelopeMetadata. 

But also packages like vochain, config, testcommon, all also import types.

Reply:
`EnvelopePackage, EnvelopeMetadata,  ScrutinizerOnProcessData` are specific for Scrutinizer, so we might move them.

`KeyKeeperMaxKeyIndex` and `VotePackage`could be moved to the vochain package.

I think with this we'll avoid circular conflicts.

Reply:
Commit https://github.com/vocdoni/vocdoni-node/commit/cbedf52fdb9f6cf0121a8fa2e3237080ee8dc40d attempts to deal with this import cycle. 

Could you check it out @p4u ? Tried to make it as clean as possible..

Reply:
and now here: https://github.com/vocdoni/vocdoni-node/commit/1affa20c8b0ea9ffcf126668587eddc814c1b3d7